calico_containers/calico_ctl/node.py

```diff
@@ -417,11 +435,11 @@ def warn_if_hostname_conflict(ip):
         else:
             if current_ipv4 != "" and current_ipv4 != ip:
                 print_paragraph("WARNING: Hostname '%s' is already in use "
-                    "with IP address %s. Calico requires each compute host to "
-                    "have a unique hostname. If this is your first time "
-                    "running 'calicoctl node' on this host, ensure that " \
-                    "another host is not already using the " \
-                    "same hostname."  % (hostname, ip))
+                                "with IP address %s. Calico requires each compute host to "
+                                "have a unique hostname. If this is your first time "
+                                "running 'calicoctl node' on this host, ensure that " \
+                                "another host is not already using the " \
+                                "same hostname."  % (hostname, ip))
```

Comment:
Nice catch here.  Can you also get rid of the unnecessary `\`'s and fit to 80 chars?